.changeset/config.json

```diff
@@ -0,0 +1,15 @@
+{
+  "$schema": "https://unpkg.com/@changesets/config@3.0.0/schema.json",
+  "changelog": "@changesets/cli/changelog",
+  "commit": false,
+  "fixed": [],
+  "linked": [],
+  "access": "public",
+  "baseBranch": "v2",
+  "updateInternalDependencies": "patch",
+  "ignore": [],
+  "privatePackages": {
+    "tag": false,
+    "version": false
+  }
```

Comment:
I didn't know about these but they prevent changeset from caring about private packages.

Reply:
Doesn't seem to work: https://github.com/microsoft/DefinitelyTyped-tools/pull/1003